optaplanner-core/src/main/java/org/optaplanner/core/api/domain/valuerange/ValueRange.java

```diff
@@ -70,7 +58,7 @@
      * Scales well because it does not require caching.
      *
      * @param workingRandom never null, the {@link Random} to use when any random number is needed,
-     *        so {@link EnvironmentMode#REPRODUCIBLE} works correctly. {@link RandomUtils} can be useful too.
+     *        so we can achieve proper reproducible environments.
```

Comment:
Avoid "we" and "I" in javadocs and docs

```suggestion
     *        so runs are reproducible when desired.
```